examples/cifar/train_cifar_custom_loop.py

```diff
@@ -0,0 +1,120 @@
+from __future__ import print_function
```

Comment:
The meaning of the word "custom loop" may not be clear for beginners, so it might be helpful if we add some docstrings that explains this code in detail. (e.g., "This code is a custom loop version of train_cifar.py. That is, we train models not using Trainer class in chainer, but we manually compute the loss of minibatches and apply an optimizer to update the models.")